2019-06-13 08:48:32,566|loggerTools.py|ERROR:error
